src/Concerns/MakesAssertions.php

```diff
@@ -176,12 +176,18 @@ public function assertDontSee($text, $ignoreCase = false)
     public function assertSeeIn($selector, $text, $ignoreCase = false)
     {
         $fullSelector = $this->resolver->format($selector);
+        $elements = $this->elements($selector) ?? [];
+//        PHPUnit::assertNotEmpty($elements);
```

Comment:
AssertNotEmpty is not strictly required because the assert below will fail, but the "exception" or reason may be different and this may not be desired